packages/compiler/server/serverlib.ts

```diff
@@ -618,9 +621,48 @@ export function createServer(host: ServerHost): Server {
   async function addImportCompletion(
     program: Program,
     document: TextDocument,
-    completions: CompletionList
+    completions: CompletionList,
+    node: StringLiteralNode
+  ) {
+    if (node.value.startsWith("./") || node.value.startsWith("../")) {
+      await addRelativePathCompletion(program, document, completions, node);
+    } else if (!node.value.startsWith(".")) {
+      await addLibraryImportCompletion(program, document, completions);
+    }
+  }
+
+  async function addRelativePathCompletion(
+    program: Program,
+    document: TextDocument,
+    completions: CompletionList,
+    node: StringLiteralNode
   ) {
-    await addLibraryImportCompletion(program, document, completions);
+    const documentPath = getPath(document);
+    const documentDir = getDirectoryPath(documentPath);
+    const nodevalueDir = getDirectoryPath(node.value);
+    const maincadl = resolvePath(documentDir, nodevalueDir);
+    const listFiles = await program.host.readDir(maincadl);
+    for (const file of listFiles) {
```

Comment:
We should also exclude from that list:
- the working file(cannot import itself)
- the `node_modules` folder 

See thje current results in the playground: Should techinically not show anything as there is a single file
![image](https://user-images.githubusercontent.com/1031227/174657334-90ed89a3-ff50-4576-b8de-246a8c4773c6.png)

Reply:
For some reason, I'm still seeing this behavior in the playground

Reply:
I think there is something wrong with readDir implementation in browser host.

Reply:
Yes, I think you need the same fix to browser-host.ts that was made to test-host.ts.

Reply:
I think you still need the change to browser-host.ts, but I'm also not seeing the bad behavior in playground now, so I'm not sure what happened.

Reply:
Actually I still see it, but I have to type "./n" before anything is completed for some reason. Nothing is completed after "./" which seems like another issue :(